### DISCUSSION TITLE
How to lock / burn liquidity on Liquidswap or another AMM DEXs?

### DISCUSSION THREAD

**Discussion by KevinTrinhUc** [aptos-labs/aptos-developer-discussions#306] [Questions]
### Describe your question in detail.

We aim to prove to our community/users that we won't rug pull. Could someone provide guidance on how to effectively lock or burn liquidity? 

### Which operating system are you using?

N/A

### Which SDK or tool are you using? (if any)

N/A

**Comment by Alexzander-Stone** [comment]
## How to lock/burn an asset
Typically, you'll see builders send unneeded assets to dead addresses when they don't own the smart contract for the asset.
Since you don't control the third party DEX, this is your best solution.

## 0xdead example
In the [Bonding Curve Launchpad example](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/bonding_curve_launchpad), liquidity tokens are sent to the [0xdead account](https://github.com/aptos-labs/aptos-core/blob/1a6d215796cc1a4555383d4e5e57a36ee6661dbe/aptos-move/move-examples/bonding_curve_launchpad/sources/liquidity_pairs.move#L363).
